src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -12,7 +12,76 @@
  */
 public class FieldXY {
 
-    public void moveRobot(Robot robot, int toX, int toY) {
+    public static void moveRobot(Robot robot, int toX, int toY) {
+        while (robot.getX() != toX) {
```

Comment:
Separate this blocks to smaller methods. It's a good practice to have some small methods instead of single and giant :)